src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -44,6 +44,17 @@ function getDeclarationErrorAddendum(owner) {
   return '';
 }
 
+//https://github.com/facebook/react/issues/5013
```

Comment:
We don't generally put links to the github issues in the source.  It clutters up the code, and it's easy enough to find the issue in github.